src/repositories/followers.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { FollowerModel } from "../models/Follower.model";
+
+export const listFollowers = (queryParams: object) => {
```

Comment:
the followers list returns a Promise. These functions should be async

Reply:
@AndrewKaenko No. We set async at setvice